src/projects.h

```diff
@@ -165,168 +167,317 @@ typedef struct { double u, v; } projUV;
+    int     (* testfunc)(void);  /* projection entry point */
 };
 
 struct PJ_ELLPS {
-    char    *id;        /* ellipse keyword name */
-    char    *major;     /* a= value */
-    char    *ell;       /* elliptical parameter */
-    char    *name;      /* comments */
+    char    *id;           /* ellipse keyword name */
+    char    *major;        /* a= value */
+    char    *ell;          /* elliptical parameter */
+    char    *name;         /* comments */
 };
 struct PJ_UNITS {
-    char    *id;        /* units keyword */
-    char    *to_meter;  /* multiply by value to get meters */
-    char    *name;      /* comments */
+    char    *id;           /* units keyword */
+    char    *to_meter;     /* multiply by value to get meters */
+    char    *name;         /* comments */
 };
 
 struct PJ_DATUMS {
-    char    *id;        /* datum keyword */
-    char    *defn;      /* ie. "to_wgs84=..." */
-    char    *ellipse_id;/* ie from ellipse table */
-    char    *comments;  /* EPSG code, etc */
+    char    *id;           /* datum keyword */
+    char    *defn;         /* ie. "to_wgs84=..." */
+    char    *ellipse_id;   /* ie from ellipse table */
+    char    *comments;     /* EPSG code, etc */
 };
 
 struct PJ_PRIME_MERIDIANS {
-    char    *id;        /* prime meridian keyword */
-    char    *defn;      /* offset from greenwich in DMS format. */
+    char    *id;           /* prime meridian keyword */
+    char    *defn;         /* offset from greenwich in DMS format. */
 };
 
-typedef struct {
-    double ll_long;      /* lower left corner coordinates (radians) */
-    double ll_lat;
-    double ur_long;      /* upper right corner coordinates (radians) */
-    double ur_lat;
-} PJ_Region;
 
 struct DERIVS {
-    double x_l, x_p;    /* derivatives of x for lambda-phi */
-    double y_l, y_p;    /* derivatives of y for lambda-phi */
+    double x_l, x_p;       /* derivatives of x for lambda-phi */
+    double y_l, y_p;       /* derivatives of y for lambda-phi */
 };
 
 struct FACTORS {
     struct DERIVS der;
-    double h, k;        /* meridinal, parallel scales */
-    double omega, thetap;   /* angular distortion, theta prime */
-    double conv;        /* convergence */
-    double s;           /* areal scale factor */
-    double a, b;        /* max-min scale error */
-    int code;           /* info as to analytics, see following */
+    double h, k;           /* meridional, parallel scales */
+    double omega, thetap;  /* angular distortion, theta prime */
+    double conv;           /* convergence */
+    double s;              /* areal scale factor */
+    double a, b;           /* max-min scale error */
+    int code;              /* info as to analytics, see following */
 };
 
+
 #define IS_ANAL_XL_YL 01    /* derivatives of lon analytic */
 #define IS_ANAL_XP_YP 02    /* derivatives of lat analytic */
-#define IS_ANAL_HK    04    /* h and k analytic */
+#define IS_ANAL_HK  04      /* h and k analytic */
```

Comment:
Well - its mostly a matter of putting less important stuff lower in the file. I've responded by moving a large group of related material (defines and the definition of the projCtx_t) down into the same block of the file.